Makefile

```diff
@@ -92,6 +92,7 @@ endif
 
 # Helper function to get dependency version from go.mod
```

Comment:
I think we now have to run the linter for every module (see the workflow in core CAPI)

If we have any unit tests in ./test the same applies there (if not already done, sorry didn't review the PR yet)